docs/framework/additional-apis/index.md

```diff
@@ -63,6 +64,8 @@ These APIs support the product infrastructure and are not intended/supported to
 * [System.Net.HttpWebRequest.\_HttpResponse Field](_httpresponse.md)
 * [System.Net.ServicePoint.m\_ConnectionGroupList Field](m_connectiongrouplist.md)
 * [System.Net.ServicePointManager.s\_ServicePointTable Field](s_servicepointtable.md)
+* [System.Net.TlsStream.m_Worker Field](system.net.tlsstream.m_worker.md)
+* [System.Net.Security.SslState.SslProtocol Property](system.net.security.sslstate.sslprotocol.md)
```

Comment:
Should these be swapped? It looks like the list is alphabetical.

Reply:
They follow the order in the TOC. It looks a bit funny but because System.Net.Security is a different namespace, so it comes separately. 
I'd have to list namespaces and then APIs to fix this here too. Not sure if it's worth since all I need is to have APIscan to detect these APIs.